### ISSUE TITLE
Issue on Upgrading

### ISSUE THREAD

**Issue by silenx** [firefly-iii/firefly-iii#2392]
Hello,
i'm trying to update manually ( i use firefly on a virtual machine ).
When i run this:
composer create-project grumpydictator/firefly-iii --no-dev --prefer-dist firefly-iii-updated 4.8.0
i get:

```
- Installing rcrowe/twigbridge (v0.9.12): Loading from cache
Generating autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan firefly:instructions install

In FireflyServiceProvider.php line 97:
                                                                            
  Class 'FireflyIII\Support\Twig\Extension\TransactionGroupTwig' not found  
                                                                            

Script @php artisan firefly:instructions install handling the post-install-cmd event returned with error code 1

```

**Comment by JC5** [comment]
Executing the exact same commands works for me. The final output is:

```
Generating optimized autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan firefly:instructions install
+------------------------------------------------------------------------------+
|                                                                              |
| Thank you for installing Firefly III, v4.8.0!                                |
| This is a huge upgrade for Firefly III. Please expect bugs and errors, and   |
| bear with me as I fix them. I tested a lot of things but pretty sure I       |
| missed some. Thanks for understanding.                                       |
|                                                                              |
+------------------------------------------------------------------------------+
> @php artisan key:generate
Application key set successfully.
```

Maybe you can clear your composer cache? Which OS are you on?

**Comment by davids3** [comment]
I'm getting a similar issue, except while following the [guide](https://docs.firefly-iii.org/en/latest/installation/server.html) to install for the first time on my virtual web server:
```
php artisan migrate:refresh --seed

In FireflyServiceProvider.php line 97:
                                                                            
  Class 'FireflyIII\Support\Twig\Extension\TransactionGroupTwig' not found  
```
I've tried clearing the composer cache. I'm using Ubuntu 16.04 with PHP 7.2.21.

**Comment by silenx** [comment]
@JC5 i'm on Ubuntu 18.04.3 LTS with php 7.2.21

**Comment by silenx** [comment]
Tryed to do the same thing with 4.8.0.1 , but issue remain ( i cleared composer cache before run 
the composer create-project grumpydictator/firefly-iii --no-dev --prefer-dist firefly-iii-updated 4.8.0.1 )

```
 - Installing rcrowe/twigbridge (v0.9.12): Downloading (100%)         
Generating autoload files
> Illuminate\Foundation\ComposerScripts::postAutoloadDump
> @php artisan firefly:instructions install

In FireflyServiceProvider.php line 101:
                                                                            
  Class 'FireflyIII\Support\Twig\Extension\TransactionGroupTwig' not found 
```

**Comment by davids3** [comment]
Retried with 4.8.0.1 as well which didn't fix it. Had a look at the source, looks like the path of the above class was wrong, it actually sits at `'FireflyIII\Support\Twig\TransactionGroupTwig'`. I changed the following files and no longer get this error:
- Line 66 of `app/Providers/FireflyServiceProvider.php` becomes `use FireflyIII\Support\Twig\TransactionGroupTwig;`
- Line 24 of `app/Support/Twig/TransactionGroupTwig.php` becomes `namespace FireflyIII\Support\Twig;`

Granted I don't know much PHP so this might just cause other errors later, @silenx are you able to comment?

**Comment by JC5** [comment]
That is very well spotted, thank you! I will include this fix in 4.8.0.2.
